flang/lib/Optimizer/CodeGen/Target.cpp

```diff
@@ -186,6 +233,15 @@ fir::CodeGenSpecifics::get(mlir::MLIRContext *ctx, llvm::Triple &trp,
       return std::make_unique<TargetX86_64>(ctx, trp, kindMap);
     }
     break;
+  case llvm::Triple::ArchType::aarch64:
+    switch (trp.getOS()) {
+    default:
+      break;
+    case llvm::Triple::OSType::Linux:
+    case llvm::Triple::OSType::Darwin:
```

Comment:
I'm assuming aarch64 Darwin is a supported LLVM target.

Reply:
I see several uses of the aarch64-apple-darwin target in llvm.
llvm-project/llvm$ grep -r aarch64-apple-darwin | wc -l
76